sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
 ## Introduction
 
-### Potential writing prompt
+Biology and medicine are rapidly becoming particularly data-intensive field with
+respect to both research and practice. A recent comparison of genomics with
+social media, online videos and other data-intensive scientific disciplines
+suggested that the field of genomics alone would equal or surpass other fields
+in data generation and analysis within the next decade
+[@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
+also new challenges. We expect that algorithms to automatically extract
+meaningful patterns and provide sufficient context to enable us to act will be
```

Comment:
Not sure how granular you'd like comments to be at this time... 

I think `act` in this sentence is a bit too vague. Perhaps something like:

_We expect that algorithms to automatically extract meaningful patterns and provide actionable knowledge allowing us to better treat, categorize, or study disease will be required._

I also think referencing the goal of the review early would set the tone of the review nicely.

I think its good to get words in the Github and not necessarily focus too much on the specific style yet so I will try to refrain from too many of these types of comments to not stall progress!